terraform/utils.go

```
@@ -456,6 +490,55 @@ func findBlockLinesFromJSON(file *os.File, blockName string, pathName ...string)
 		startLine = 0
 	}
 
+	// By default (when created), the file content is not properly formatted with indentation and all the content remains in line 1
+	if file != nil && startLine == 0 && endLine == 0 {
+		// Set the start line as 1, and
+		// end line as the current line (i.e. total lines)
+		startLine = 1
+		endLine = currentLine
+
+		content, _ := os.ReadFile(path)
```

We shouldn't ignore the error if this function can return one